backend/capellacollab/sessions/files/routes.py

```diff
@@ -61,29 +53,29 @@ def get_files(
 
 
 @router.post("/", dependencies=[Depends(check_session_belongs_to_user)])
```

Comment:
`check_session_belongs_to_user` can be removed globally if `get_current_session` is used everywhere.

Reply:
Done